cli/src/helper.rs

```diff
@@ -343,3 +343,29 @@ pub async fn check_file_exists(path: &Path) -> Result<(), Error> {
     }
     Ok(())
 }
+
+pub enum SecretManagerChoice {
+    Stronghold,
+    LedgerNano,
+}
+
+impl From<usize> for SecretManagerChoice {
+    fn from(value: usize) -> Self {
+        match value {
+            0 => Self::Stronghold,
+            1 => Self::LedgerNano,
```

Comment:
What do you think about adding LedgerNanoSimulator here? Or would another place be better to select this option?
Because I think that would be very good if we can actually use a simulator as well with it for testing (so need to create `LedgerSecretManager::new(true)`)

Reply:
I like the idea. Will add it.

Reply:
Added it ... I think ... Is that what you wanted?

Reply:
Yes that's good